index.js

```diff
@@ -422,3 +424,15 @@ class Event {
     }
   }
 }
+
+class Session {
+  constructor (listeners) {
+    this.listeners = listeners;
```

Comment:
```suggestion
    this._listeners = listeners;
```

To make it clear that this isn't public API.